polkadot-parachains/src/command.rs

```diff
@@ -68,11 +68,11 @@ fn load_spec(
 		"encointer-kusama" 			=> Box::new(chain_spec::launch_kusama()?),
 
 		// live config initialize
-		"encointer-rococo-fresh" 		=> Box::new(chain_spec::launch_spec(
+		"rococo-fresh" 		=> Box::new(chain_spec::launch_spec(
```

Comment:
why not `launch-rococo`? Would be more in line with other namings. The word fresh means nothing else than being the foundation of a new live chain, which needs to be launched. so launch seems right to me.
But then we should refactor the helper fn `launch-*()` which supplies the hard-coded spec to `live-*()`

Reply:
I'll leave it a t your suggestion. Every possible variant can be confusing in its own way